samples/cpp/benchmark_app/.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
suggesting to remove this, this file is not necessary to put under source control